examples/tensorflow/segmentation/models/maskrcnn_model.py

```diff
@@ -37,7 +36,7 @@ def _restore_baseline_weights(keras_model, checkpoint_path):
 
     assignment_map = {}
     for v in keras_model.variables:
-        var_name = v.name[v.name.find('/') + 1:] # Skip parent name
+        var_name = v.name
```

Comment:
Could you please confirm that the baseline checkpoint was successfully restored?

Reply:
Thanks for the notice! 
Yes, I confirm that the baseline checkpoint is restored.